examples/preload-aliases.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+/**
+ * If you need to run the Requests v2.x library with another framework/CMS that still ships v1.x,
+ * use the following to preload the class aliases before the actual implementations of Requests v1.x. get loaded.
```

Comment:
```suggestion
 * use the following to preload the class aliases before the actual implementations of Requests v1.x. gets loaded.
```

Reply:
If the verb is singular, the noun should then also be, right; i.e. `the actual implementation … gets`?

Reply:
Correct.

Reply:
Oh, right, I misead.